service/deployer/installer/helm/helm.go

```diff
@@ -219,14 +219,15 @@ func (i *HelmInstaller) Install(event eventerspec.DeploymentEvent) error {
 		return microerror.MaskAny(err)
 	}
 
-	if err := i.runHelmCommand(
-		"install",
-		"install",
-		tarballPath,
+	installCommand := []string{
+		"upgrade",
```

Comment:
`upgrade --install` essentially creates the release if it's not already installed, or upgrades it if it is. Nice.